plugins/events-correlation-engine/src/main/java/org/opensearch/plugin/correlation/events/resthandler/RestSearchCorrelationAction.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.plugin.correlation.events.resthandler;
+
+import org.opensearch.client.node.NodeClient;
+import org.opensearch.rest.BaseRestHandler;
+import org.opensearch.rest.RestRequest;
+
+import java.io.IOException;
+
+public class RestSearchCorrelationAction extends BaseRestHandler {
```

Comment:
nit : Should it be ``RestSearchCorrelatedEventsAction`` to keep it consistent with the related Actions? I am open to both

Reply:
renamed it.